lib/units/test/unit_test.cpp

```diff
@@ -31,6 +31,15 @@ TEST(UnitTest, construct_bad_string) {
   EXPECT_THROW(Unit("abcde"), except::UnitError);
 }
 
+TEST(UnitTest, custom_unit_strings_get_rejected) {
+  // Custom (counting) units and equation units are rejected.
+  for (const auto &str : {"CXUN[0]", "CXUN[51]", "CXUN[1023]", "CXCUN[0]",
+                          "CXCUN[15]", "decibels", "ln(x)", "EQXUN[1]",
```

Comment:
How did `decibels` work previously? I suppose Scipp was completely broken with unit conversion and arithmetic ops?

Reply:
I think conversions didn't work, e.g.
```.py
(1.2*sc.Unit('decibel')).to(unit='m decibel')
```
raises
```
UnitError: Conversion from `EQXUN[3]` to `0.001*EQXUN[3]` is not valid.
```
(it is an equation unit, FYI)